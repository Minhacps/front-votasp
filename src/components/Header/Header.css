.header {
  background-color: var(--roxo);
  display: flex;
  flex-direction: row-reverse;
  padding: 15px;
  align-items: center;
  justify-content: flex-end;
}

.logo {
  width: 130px;
}

@media (min-width: 800px) {
  .header {
    flex-direction: row;
    justify-content: space-between;
  }
}
