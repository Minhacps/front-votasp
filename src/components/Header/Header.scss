.header {
  background-color: var(--roxoClaro);
  background-color: #7e798c;
  padding: 15px;
}
