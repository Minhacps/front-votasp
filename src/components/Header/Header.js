import React from 'react';
import './Header.scss';

import logoUrl from '../../img/logo-votasp.png';

const Header = () => {
  return (
    <header className="header">
      <img src={logoUrl} alt="Logotipo escrito Vota em branco e SP em amarelo." />
    </header>
  );
};

export default Header;
