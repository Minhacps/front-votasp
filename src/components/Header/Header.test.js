import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './Header';

describe('<Header />', () => {
  it('should match snapshot', () => {
    const header = shallow((
      <Header />
    ))

    expect(toJson(header)).toMatchSnapshot();
  });
});
