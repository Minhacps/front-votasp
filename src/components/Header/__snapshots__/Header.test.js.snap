// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<header
  className="header"
>
  <img
    alt="Logotipo escrito Vota em branco e SP em amarelo."
    src="test-file-stub"
  />
</header>
`;
